File: frontend/olympic_it_project/olympic_it_project/lib/dto/admin_manager/classes/update_class_request.dart
class UpdateClassRequest {
  final String className;
  final int academicYearId;

  UpdateClassRequest({
    required this.className,
    required this.academicYearId,
  });

  Map<String, dynamic> toJson() {
    return {
      "className": className,
      "academicYearId": academicYearId,
    };
  }
}

File: backend/olympic_it_project/src/main/java/org/example/olympic_ot_project/dto/classes/UpdateClassRequest.java
package org.example.olympic_ot_project.dto.classes;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import lombok.Data;

@Data
public class UpdateClassRequest {
    @NotBlank(message = "Tên lớp không được để trống")
    private String className;
    @NotNull(message = "Năm học không được để trống")
    private Integer academicYearId;
}
